### DISCUSSION TITLE
I'm back after some time of with our new 👶

### DISCUSSION THREAD

**Discussion by tomkerkhove** [tomkerkhove/promitor#2041] [Announcements]
As you may have noticed, it's been almost 2 months since the last release and there was not a lot of movement.

That is because I became a father again in mid-March and have been busy lately but I'm back! I have been working on the plan for the next release which you will see starting to get shape very soon.

Sorry about that!

**Comment by MattiasAng** [comment]
Congratulations! 🎉

**Reply by tomkerkhove** [reply]
Thank you!
